content/en/tutorials/elb-load-balancing/index.md

```diff
@@ -319,8 +321,10 @@ $ curl http://lb-test-1.elb.localhost.localstack.cloud:4566/hello2 | jq
 "Hello 2"
 {{< / command >}}
 
+Both commands send an HTTP GET request to the endpoints and uses `jq` to format the response. The expected outputs are `Hello 1` & `Hello 2`, representing the Lambda functions' response.
+
 ## Conclusion
 
-This tutorial shows how to create an Application Load Balancer with two Lambda functions as targets using LocalStack. We have also seen how to create, configure, and deploy a Serverless project with LocalStack serving as an emulated local AWS environment that allows you to develop and test your Cloud & Serverless applications with AWS locally. 
+In this tutorial, we have learned how to create an Application Load Balancer (ALB) with two Lambda functions as targets using LocalStack. We have also explored creating, configuring, and deploying a Serverless project with LocalStack. This enables developers to develop and test Cloud and Serverless applications locally conveniently.
 
-LocalStack also offers integrations with other popular tools such as Terraform, Pulumi, Serverless Application Model (SAM), and more. You can find more information about LocalStack integrations on our [Integration documentation]({{< ref "user-guide/integrations" >}}). You can find the code for this tutorial (including a `Makefile` to execute it step-by-step) in our [LocalStack Pro samples over GitHub](https://github.com/localstack/localstack-pro-samples/tree/master/elb-load-balancing).
+LocalStack offers integrations with various popular tools such as Terraform, Pulumi, Serverless Application Model (SAM), and more. For more information about LocalStack integrations, you can refer to our [Integration documentation]({{< ref "user-guide/integrations">}}). To further explore and experiment with the concepts covered in this tutorial, you can access the code and resources on our [LocalStack Pro samples over GitHub](https://github.com/localstack/localstack-pro-samples/tree/master/elb-load-balancing) along with a `Makefile` for step-by-step execution.
```

Comment:
suggestions (for tutorials in general): it would be nice to have a section at the top, something like `resources` or `source-code` so that people know at the beginning that there is a sample project as well.